SU-HSDO/suhumsci
# SHS-5810: Text "Main content start" appears after clicking "Skip to main content"

# READY FOR REVIEW

## Summary
Fix main content start text after clicking skip to main content

## Need Review By (Date)
09/25

## Urgency
medium

## Steps to Test
1. Go to the homepage on Traditional or Colorful
2. Using a screen reader or only the keyboard
3. Press `Tab`
4. You will see the "Skip to main content".  Press `Enter`
5. Verify the text "Main content start" is not visible anymore

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208403963078981
